app/helpers/manage_metadata_helper.rb

```diff
@@ -1,12 +1,15 @@
 # :nodoc:
 module ManageMetadataHelper
   def display_entry_id(metadata, type)
-    blank_short_name = type == 'draft' ? '<Blank Short Name>' : 'New Collection'
-    short_name = metadata['ShortName'] || blank_short_name
+    short_name = if type.include? 'collection'
```

Comment:
sneaky -- `include` instead of `replace`. 🎩 (may want to downcase it, unless we're sure its always lower case as `include` is case sensitive)